codegenerator/cli/templates/dynamic/codegen/src/TestHelpers_MockDb.res.hbs

```diff
@@ -329,7 +329,7 @@ and makeProcessEvents = (mockDb: t, ~chainId=?) => async (
     ) {
       registrations
     } else {
-      Generated.registerAllHandlers()
+      await Generated.registerAllHandlers()
```

Comment:
I seem to remember if we made this async it causes problems for the testing framework. Does that still work fine?

Reply:
Yes, I changed the testing framework some time ago to support this case